tests/build-rss.test.ts

```diff
@@ -1,10 +1,11 @@
-const fs = require('fs');
-const path = require('path');
-const { XMLParser } = require('fast-xml-parser');
-const { rssFeed } = require('../scripts/build-rss.ts');
+import { XMLParser } from 'fast-xml-parser';
+import fs from 'fs';
+import path from 'path';
+
+import { rssFeed } from '../scripts/build-rss';
 
 const parser = new XMLParser({ ignoreAttributes: false });
-const { mockRssData, title, type, desc, missingDateMockData, incompletePostMockData } = require('./fixtures/rssData');
+const { mockRssData, title, type, desc, incompletePostMockData } = require('./fixtures/rssData');
```

Comment:
Why this const is not converted to ESM import?

Reply:
Done!